IPython/utils/tests/test_text.py

```diff
@@ -180,6 +180,7 @@ def test_LSString():
     lss = text.LSString("abc\ndef")
     nt.assert_equal(lss.l, ['abc', 'def'])
     nt.assert_equal(lss.s, 'abc def')
+    nt.assert_in("path", sys.modules)
```

Comment:
Actually, it should be OK with `abc` and `def` - they're valid filenames, and there's no requirement that a path refers to a file that actually exists.

The main purpose of the exercise is a *smoketest*: run some code and check that it doesn't error. But we know that the return values should be `path.path` objects, so you could check like this:

```python
import path
...
def test_LSString():
    ...
    nt.assert_isinstance(lss.p[0], path.path)
```